packages/puppeteer-extra-plugin-devtools/lib/RemoteDevTools.js

```diff
@@ -100,7 +100,8 @@ class DevToolsTunnel extends DevToolsCommon {
     return {
       prefix: 'devtools-tunnel',
       subdomain: null,
```

Comment:
This is not even documented and should be deprecated in favor of passing it in the `localtunnel` object.